Hunting Queries/TeamsLogs/MultipleTeamsDeletes.yaml

```diff
@@ -5,10 +5,6 @@ description: |
   This query requires you to have Teams data collected from the O365 Management Activity API
   and the Teams parser function enabled.
   More details: https://techcommunity.microsoft.com/t5/azure-sentinel/protecting-your-teams-with-azure-sentinel/ba-p/1265761'
-requiredDataConnectors:
```

Comment:
We will want to flip this back to Office Activity once Teams data is fully supported there.